libethereum/Precompiled.cpp

```diff
@@ -699,7 +699,10 @@ ETH_REGISTER_PRECOMPILED( logTextMessage )( bytesConstRef _in ) {
 }
 
 static const std::list< std::string > g_listReadableConfigParts{ "skaleConfig.sChain.nodes.",
```

Comment:
I think this variable was required to use in precompiled contracts that we have disabled already (like `getConfigVariableU256` and etc). so we don't need it anymore